ggml/include/ggml-backend.h

```diff
@@ -331,7 +331,7 @@ extern "C" {
     };
 
     // Copy a graph to a different backend
-    GGML_API struct ggml_backend_graph_copy ggml_backend_graph_copy(ggml_backend_t backend, struct ggml_cgraph * graph);
+    GGML_API struct ggml_backend_graph_copy ggml_backend_graph_copy_init(ggml_backend_t backend, struct ggml_cgraph * graph);
```

Comment:
@slaren This function caused a shadow warning with GCC, so I renamed it to `ggml_backend_graph_copy_init`:

```
llama.cpp/ggml/src/../include/ggml-backend.h:334:119: warning: ‘ggml_backend_graph_copy ggml_backend_graph_copy(ggml_backend_t, ggml_cgraph*)’ hides constructor for ‘struct ggml_backend_graph_copy’ [-Wshadow]
  334 |     GGML_API struct ggml_backend_graph_copy ggml_backend_graph_copy(ggml_backend_t backend, struct ggml_cgraph * graph);
```

Reply:
I would rather rename the struct to something like `ggml_backend_graph_copy_state`, since this is not really a constructor, it's just that it returns multiple objects to the caller.